include/clang/Basic/TokenKinds.def

```diff
@@ -571,8 +571,12 @@ KEYWORD(__bridge_retained            , KEYARC)
 KEYWORD(__bridge_retain              , KEYARC)
 
 
-// CHERI casts to and from capabilities
-KEYWORD(__cheri_cast                 , KEYALL)
+// CHERI casts.
+KEYWORD(__cheri_ptr                 , KEYALL) // cast ptr <-> capability
```

Comment:
I don't really like using `__cheri_ptr` to create capabilities: `(__cheri_ptr void* __capability)ptr;`?

How about adding `__cheri_cap` or maybe `__cheri_to_ptr`+`__cheri_to_cap`. Are there any cases where we need the current `__cheri_cast` sematics of converting either pointer -> cap or cap -> pointer? I don't think there were any in CheriBSD that did conversions both ways.

Reply:
After talking with theraven, he suggested to add `__cheri_tocap` and `__cheri_fromcap`